tests/codebuild/run_integtest.sh

```diff
@@ -113,22 +114,55 @@ tar -xf sagemaker-k8s-operator.tar.gz
 pushd sagemaker-k8s-operator
 # Setup the PATH for smlogs
     mv smlogs-plugin/linux.amd64/kubectl-smlogs /usr/bin/kubectl-smlogs
-    pushd sagemaker-k8s-operator-install-scripts
-        # Since OPERATOR_AWS_SECRET_ACCESS_KEY and OPERATOR_AWS_ACCESS_KEY_ID defined in build spec, we will not create new user
-        ./setup_awscreds
-        path_to_installer=$(pwd)
-    popd
+    path_to_installer=$(pwd)/sagemaker-k8s-operator-install-scripts
 popd
 
+# A helper function to generate an IAM Role name for the current cluster and sepcified namespace
+# Parameter:
+#    $1: Namespace of CRD
+function generate_iam_role_name {
+    local crd_namespace="$1"
+    local cluster=$(echo ${cluster_name} | cut -d'/' -f2)
+
+    role_name="${cluster}-${crd_namespace}"
+    # IAM Role name must have length less than 64
+    role_name=`echo $role_name|cut -c1-64`
+}
+
+# A helper function that generates the namespace-scoped operator installer yaml file with updated namespace and role. 
+# Parameter:
+#    $1: Namespace to which the op pod is installed. For cluster scope, this value is not used. 
+#    $2: Path to the kustomize source
+#    $3: Name of the IAM Role to use
+# TODO:  Investigate if it is possible to overlay values when we build the Kustomize targets instead. 
+function generate_operator_installer_for_given_role {
+    local crd_namespace="$1"
+    local file_path="$2"
+    local role="$3"
+    local aws_account=$(aws sts get-caller-identity --query Account --output text)
+
+    kustomize build "${file_path}" > temp_file.yaml
+    sed -i "s/PLACEHOLDER-NAMESPACE/$crd_namespace/g" temp_file.yaml
+    sed -i "s/123456789012/$aws_account/g" temp_file.yaml
+    sed -i "s/DELETE_ME/$role/g" temp_file.yaml
+}
+
+
 echo "[SECTION] Run integration tests for the cluster scoped operator deployment"
+
+generate_iam_role_name "${default_operator_namespace}"
+cd scripts && ./generate_iam_role.sh "${cluster_name}" "${default_operator_namespace}" "${role_name}" "${cluster_region}" && cd ..
```

Comment:
nit: Why do we need the `cd` for this? Could we not simply use the larger path `scripts/generate_iam_role.sh`?

Reply:
yea, just carried forward from old code, cleaning.